src/v0/destinations/sendinblue/data/SendinbluePageConfig.json

```
@@ -0,0 +1,39 @@
+[
+  {
+    "destKey": "email",
+    "sourceKeys": "emailOnly",
+    "sourceFromGenericMap": true
+  },
+  {
+    "destKey": "ma_title",
```

```suggestion
    "destKey": "properties.ma_title",
```
We can also provide as above.

---

Do i need to add this mapping for `path` , `referrer` as well?
i.e. 
`properties.ma_referrer, properties.ma_path`

---

Yes, I want to say we support multilevel mapping as well so wherever possible we can use that.

---

added the above mapping for ma_title, ma_path, ma_referrer